static/test/javascripts/spec/common/adblock-banner.spec.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'common/modules/commercial/adblock-banner-config',
+    'common/modules/adblock-banner'
+], function (adblockBannerConfig, AdblockBanner) {
+
+    describe('Adblock banner', function () {
+
+        var ukBanners;
+        beforeEach(function () {
+            document.innerHtml = '<div class="top-banner-ad-container--desktop"></div>';
+            ukBanners = adblockBannerConfig.getBanners('UK');
+        });
+
+        it('should be able to render every type of banner', function () {
```

Comment:
Ah, so it'd be a kind of integration test? You'd be looking to make sure that none of the template variables were replaced with `undefined`s.

If you do that, you might find it easier to test not the result of the render, but the result of the function that will be passed to fastdom. This would make testing a lot easier, because otherwise you have to unit test an asynchronous function and give it a way of telling the test framework that it's finished.